__tests__/app.test.js

```diff
@@ -1,225 +1,319 @@
+        },
+        {
+          'place_id': '312432792',
+          'licence': 'https://locationiq.com/attribution',
+          'osm_type': 'way',
+          'osm_id': '165271257',
+          'boundingbox': [
+            '14.6696649',
+            '14.6703081',
+            '121.0988688',
+            '121.0994135'
+          ],
+          'lat': '14.6703081',
+          'lon': '121.0994135',
+          'display_name': 'Seattle, Vista Real Classica, Batasan Hills, 2nd District, Quezon City, Eastern Manila District, Metro Manila, 1808, Philippines',
+          'class': 'highway',
+          'type': 'service',
+          'importance': 0.175
+        },
+        {
+          'place_id': '6534059',
+          'licence': 'https://locationiq.com/attribution',
+          'osm_type': 'node',
+          'osm_id': '668442224',
+          'boundingbox': [
+            '47.6028456',
+            '47.6029456',
+            '-122.3398908',
+            '-122.3397908'
+          ],
+          'lat': '47.6028956',
+          'lon': '-122.3398408',
+          'display_name': 'Seattle, Colman Dock, West Edge, Belltown, Seattle, King County, Washington, 98104, USA',
+          'class': 'amenity',
+          'type': 'ferry_terminal',
+          'importance': 0.101
+        }
+      ];
 
-    const actual = wrangleLocationResponse(response);
+      const actual = wrangleLocationResponse(response);
       
-    expect(actual).toEqual(expectation);
+      expect(actual).toEqual(expectation);
+    });
+
+    test('returns weather data', async() => {
+
+      const expectation = [
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) },
+        { forecast: expect.any(String), time: expect.any(String) }
+      ];
+  
+      const data = await fakeRequest(app)
+        .get('/weather?latitude=47.6038321&longitude=-122.3300624')
+        .expect('Content-Type', /json/)
+        .expect(200);
+
+    
+      expect(data.body).toEqual(expectation);
+    });
+
+    test('returns weather data', async() => {
```

Comment:
These two weather tests are nearly duplicates of each other - you can safely remove one or the other. Since the forecast might not stay the same, I'd be inclined to remove the second test.